requirements.txt

```diff
@@ -4,18 +4,17 @@ joblib==0.11
 matplotlib==2.1.2
 nose==1.3.7
 notebook==5.7.2
-numpy==1.14.*
+numpy==1.14.0
 pandas==0.23.4
```

Comment:
Do things break if we use the latest version of `pandas`?

Reply:
Not sure. I'll test it out. :)

Reply:
Yes, they do.